scripts/orama/sync-orama-cloud.mjs

```diff
@@ -0,0 +1,58 @@
+import { siteContent } from './get-documents.mjs';
+
+const INDEX_ID = process.env.ORAMA_INDEX_ID;
+const API_KEY = process.env.ORAMA_SECRET_KEY;
+const oramaAPIBaseURL = `https://api.oramasearch.com/api/v1/webhooks/${INDEX_ID}`;
+
+const oramaHeaders = {
+  'Content-Type': 'application/json',
+  Authorization: `Bearer ${API_KEY}`,
+};
+
+async function runUpdate() {
```

Comment:
It'd be great as mentioned before, to have inline doc blocks with reference to Orama API docs and etc.

Reply:
Yes, still work in progress but I'll document everything 🙏